src/app/components/applications/SSV/RegisterValidatorHome/components/OfflineKeyShareGeneration/OfflineKeyShareGeneration.styles.ts

```diff
@@ -4,12 +4,12 @@ import { Theme } from '@mui/material/styles';
 const text = {
   fontSize: 16,
   fontWeight: 500,
-  lineHeight: 1.62,
+  lineHeight: 1.62
```

Comment:
if you do changes to mui styles file - let's move it to styled-components in the component itself and remove this